coalib/processes/Processing.py

```diff
@@ -443,8 +444,9 @@ def yield_ignore_ranges(file_dict):
         stop_ignoring = False
         for line_number, line in enumerate(file, start=1):
             # Before lowering all lines ever read, first look for the biggest
-            # common substring, case sensitive: I*gnor*e, start i*gnor*ing.
-            if 'gnor' in line:
+            # common substring, case sensitive: I*gnor*e, start i*gnor*ing
+            # and for N*oqa start n*oqa
+            if 'gnor' or 'oqa' in line:
```

Comment:
or depending on operator priority it could also ignore your changes, as `'gnor' or 'oqa'` evaluates to `'gnor'` which shall be searched in `line`, same behaviour like before